mantidproject/mantid
# MuscatSofQW algorithm

Fixes #13243 

To test:
- Run the algorithm as per the usage example
- Set the value of EnergyMax and see that that is the limit of the X axis in the result data

The other algorithm to go with this is not complete yet so no release notes updates for now.
